packages/components/carousel/src/use-carousel.ts

```diff
@@ -192,6 +204,12 @@ export const useCarousel = (
     startTimer()
   }
 
+  function setContainerHeight(height: number) {
+    if (props.height === 'auto') {
+      containerHeight.value = height
+    }
+  }
```

Comment:
**Non-blocking:** For my personal preference, I would write this in this way:

```suggestion
  function setContainerHeight(height: number) {
    if (props.height !== 'auto') return
    containerHeight.value = height
  }
```

For me early return would make the code more readable.

Reply:
Thank you, let me modify it